packages/@sanity/server/src/configs/webpack.config.js

```diff
@@ -87,11 +87,10 @@ export default (config = {}) => {
           use: {
             loader: resolve('babel-loader'),
             options: babelConfig || {
-              presets: [resolve('babel-preset-react'), resolve('babel-preset-es2015')],
+              presets: [resolve('@babel/preset-react'), resolve('@babel/preset-env')],
```

Comment:
Where are the env targets for the studio specified?

Reply:
They are not specified, which makes it behave like preset-es2015 like before. It may be a good idea to specify, tough. What should it be?

Reply:
What we're using generally is:
```
{
  "node": "6",
  "chrome": "59",
  "safari": "10",
  "firefox": "56",
  "edge": "14"
}
```